test/spec/features/create/CreateSpec.js

```diff
@@ -483,6 +483,34 @@ describe('features/create - Create', function() {
       }
     ));
 
+
+    it('should pass hints to commandStack.shape.create event context', inject(
+      function(create, eventBus, dragging) {
+
+        // given
+        var spy = sinon.spy(function(event) {
+          expect(event.context.hints.foo).to.exist;
+        });
+
+        eventBus.once('create.start', 0, function(event) {
+          event.context.hints.foo = 'foo';
+        });
+
+        eventBus.once('commandStack.shape.create.execute', 0, spy);
```

Comment:
priority is an optional argument, you do not have to pass `0`. Or does it actually make a difference here?